packages/bpk-docs/src/pages/NativeCardPage/NativeCardPage.js

```diff
@@ -0,0 +1,99 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import readme from './../../../../../native/packages/react-native-bpk-component-card/readme.md';
+
+import iosScreenshotDefault from './../../../../../native/packages/react-native-bpk-component-card/screenshots/ios/default.png';
+import androidScreenshotDefault from './../../../../../native/packages/react-native-bpk-component-card/screenshots/android/default.png';
+import iosScreenshotWithoutPadding from './../../../../../native/packages/react-native-bpk-component-card/screenshots/ios/without-padding.png';
+import androidScreenshotWithoutPadding from './../../../../../native/packages/react-native-bpk-component-card/screenshots/android/without-padding.png';
+
+
+import Paragraph from './../../components/Paragraph';
+import DocsPageBuilder from './../../components/DocsPageBuilder';
+
+
+const components = [
+  {
+    id: 'default',
+    title: 'Default',
+    blurb: 'The default card comes with padding already applied to get you up and running quickly and has normal and hover states baked in.',
+    screenshots: [
+      {
+        title: 'iOS',
+        width: 750,
+        height: 1334,
+        src: `/${iosScreenshotDefault}`,
+        altText: 'iOS Default Card Component',
+        subText: '(iPhone 6 Simulator)',
```

Comment:
I think we wanna standardise on the emulators we are generating screenshots on (iphone 8 in this case)